src/guiapi/src/screen.c

```diff
@@ -3,9 +3,9 @@
 #include "screen.h"
 #include "gui.h"
 
-#define SCREEN_MAX_SCREENS 32
+#define SCREEN_MAX_SCREENS 33
```

Comment:
increasing MAX_SCREENS is not necessary in this PR

Reply:
opraveno zpet na 32